transcoder/__init__.py

```diff
@@ -17,26 +17,240 @@
 # limitations under the License.
```

Comment:
For posterity, this should probably be a rename and modification of MessageParser.py